### DISCUSSION TITLE
How to change only loading screen logo

### DISCUSSION THREAD

**Discussion by tttigo** [mozilla/hubs#3574] [Hubs Cloud Questions] [answered]
Hey.

I'd like to change only the logo image of loading screen, regardless of the main logo.

Via admin panel, if I change the homepage logo, to loading screen also changes...

How can I do this?

Thanks!

**Comment by robinkwilson** [comment, marked as answer]
This would be possible with [deploying a custom client](https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html) by changing the image in the code (the image in: https://github.com/mozilla/hubs/blob/hubs-cloud/src/react-components/loader.js).

AND/OR

**Open an issue/feature request in hubs-cloud repo (https://github.com/mozilla/hubs-cloud/issues)**! I'm sure you're not alone in wanting this feature in the admin panel. Link the issue here and thumbs up the issue to vote for priority!

**Reply by robinkwilson** [reply]
Marking as answered, please "unmark" and add more information if we need to revisit.
